lib/typeform_data/requestor.rb

```diff
@@ -4,6 +4,12 @@
 module TypeformData
   module Requestor
 
+    RETRY_EXCEPTIONS = [
+      Net::HTTPServiceUnavailable,
```

Comment:
Just making sure, since I couldn't find documentation on it; does `Net::HTTP` indeed throw this error in the case of a 503, rather than returning a response of a certain type as it does with e.g. 404 errors?